app/src/main/java/ca/klapstein/baudit/Data/Record.java

```diff
@@ -5,6 +5,21 @@
  *
  * @see Problem
  */
-public class Record {
+public class Record implements Comparable<Record> {
     private static final String TAG = "Record";
+
+    private long timestamp;
+
+    public Record() {
+        this.timestamp = System.currentTimeMillis();
```

Comment:
use a `Date` object instead

also look at https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html for converting a `Date` into a `String`

Reply:
implemented Date instead